### PR TITLE
Add non maple STM32F103CB_malyan build env.

### PR DESCRIPTION
**Pull request by ellensp**
### Description

Add non maple STM32F103CB_malyan

### Requirements

BOARD_MALYAN_M200

### Benefits

Less maple

### Notes
I can't get original maple STM32F103CB_malyan_maple to build at all, but have left it in there.

### CODE REVIEW

File: ini/stm32f1.ini [hunk 1]
```diff
@@ -282,3 +282,14 @@ build_unflags        = ${common_stm32.build_unflags} -DUSBCON -DUSBD_USE_CDC
...
   pre:buildroot/share/PlatformIO/scripts/generic_create_variant.py
   buildroot/share/PlatformIO/scripts/stm32_bootloader.py
+
+#
+# Malyan M200 (STM32F103CB)
+#
+[env:STM32F103CB_malyan]
+platform    = ${common_stm32.platform}
+extends     = common_stm32
+board       = malyanm200_f103cb
```
Comment (rhapsodyv) [inline ini/stm32f1.ini:292]: Which variant this board will use?

Most of the stm32duino variants can’t be used in marlin without minor changes.

Comment (ellensp) [reply ini/stm32f1.ini:292]: only used by BOARD_MALYAN_M200

Comment (ellensp) [reply ini/stm32f1.ini:292]: malyanm200_f103cb is part of platformio provided boards https://docs.platformio.org/en/latest/boards/ststm32/malyanm200_f103cb.html

Comment (rhapsodyv) [reply ini/stm32f1.ini:292]: And the variant?

we need to make sure the pins in variant.h file are all compile-time constants. That is the main reason we copy variants to marlin.

**Comment by rhapsodyv** [comment]
I checked. It uses the stm32duino variant `MALYANM200_F103CB`, and it looks 100% compatible with marlin needs, so we can merge this.

**Pull request merged**